18-minimal/Dockerfile.c8s

```diff
@@ -0,0 +1,68 @@
+FROM ubi8/ubi-minimal:latest
+
+EXPOSE 8080
+
+# Add $HOME/node_modules/.bin to the $PATH, allowing user to make npm scripts
+# available on the CLI without using npm's --global installation mode
+# This image will be initialized with "npm run $NPM_RUN"
+# See https://docs.npmjs.com/misc/scripts, and your repo's package.json
+# file for possible values of NPM_RUN
+# Description
+# Environment:
+# * $NPM_RUN - Select an alternate / custom runtime mode, defined in your package.json files' scripts section (default: npm run "start").
+# Expose ports:
+# * 8080 - Unprivileged port used by nodejs application
+ENV APP_ROOT=/opt/app-root \
+    # The $HOME is not set by default, but some applications need this variable
+    HOME=/opt/app-root/src \
```

Comment:
This can be rewritten to variable too. https://github.com/sclorg/s2i-base-container/blob/master/core/Dockerfile.c8s#L25

Reply:
I don't understand what you mean by this, can you explain more, pls?

Reply:
Same reasoning as before - we are not using s2i-core in minimal images, hence we cannot use the variables the image provides and need to set our own.

Reply:
ok, so marking this as solved